websurfx/config.lua

```diff
@@ -73,3 +73,5 @@ upstream_search_engines = {
     Mojeek = false,
     Bing = false,
 } -- select the upstream search engines from which the results should be fetched.
+
+proxy = "" -- Proxy to send outgoing requests through. Set to empty string to disable.
```

Comment:
```suggestion
proxy = "" -- Proxy to send outgoing requests through. Set to empty string to disable.
```
```suggestion
proxy = nil -- Proxy to send outgoing requests through. Set to nil to disable.
```

As suggested previously, I would suggest having the value for this as nil which better explains/documents the user whether the option is disabled or enabled. 🙂